xonsh/lexer.py

```diff
@@ -408,10 +407,39 @@ def _new_token(type, value, pos):
     return o
 
 
+def _get_token_types():
+    kwlist = kwmod.kwlist[:]
+    if PYTHON_VERSION_INFO >= (3, 9, 0) and PYTHON_VERSION_INFO < (3, 10):
+        kwlist.remove("__peg_parser__")
```

Comment:
This can be simply `version == 3.9` ?

Reply:
Yes, simply `sys.version_info[:2] == (3, 9). Note that I didn't write this code - it was just moved. I don't know why we have our own variable for this.